app/components/AvailabilityBox/index.js

```diff
@@ -0,0 +1,69 @@
+/**
+*
+* Availability
+*
+*/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import StyledSmall from 'components/shared/StyledSmall';
+import PageTitle from 'components/shared/PageTitle';
+import moment from 'moment';
+import {} from 'react-intl';
+
+const AvailabilityHeader = styled.h2 `
```

Comment:
Sorry to do this but can you remove all this font- stuff, we'll revisit this later but I think we should avoid all custom font stuff on a per-component basis